docs/howto/openstack/magnum/new-k8s-cluster.md

```diff
@@ -0,0 +1,308 @@
+# Creating new Kubernetes clusters
+
+By employing OpenStack 
+[Magnum](https://docs.openstack.org/magnum/latest) you can easily 
+create Kubernetes clusters over OpenStack, using either the {{gui}} or 
+the OpenStack CLI. Let us demonstrate the creation of a Kubernetes 
+cluster following both approaches.
+
+## Prerequisites
+
+First and foremost, you need an [account in {{brand}} 
+Cloud](/howto/getting-started/create-account). If you prefer to work 
+with the OpenStack CLI, go ahead and [enable it 
+first](/howto/getting-started/enable-openstack-cli). Then, make sure 
+you have the appropriate Magnum client Python module installed. Use 
+either the package manager of your operating system or `pip`:
+
+=== "Debian/Ubuntu"
+	```bash
+	apt install python3-magnumclient
+	```
+=== "Mac OS X with Homebrew"
+	This Python module is unavailable via `brew`, but you can have 
```

Comment:
I'd suggest to replace "have" with "install".

Reply:
Good suggestion!